spec/models/collection_spec.rb

```diff
@@ -4,96 +4,104 @@
 
 RSpec.describe Collection do
   let(:user) { create(:user) }
-  let(:collection) { described_class.create(title: collection_title_fixture, user:) }
+  let(:collection) { create(:collection, title: collection_title_fixture, user:) }
```

Comment:
Prefer factories.

Reply:
Yes, but this is a test of the Collection class so I'd suggest it should be tested directly instead of via a factory.

Reply:
@justinlittman `FactoryBot.create(:collection)` does create an instance of the `Collection` class and it doesn't seem to have drawbacks. And it gets us FactoryBot goodness like automagic clean-up.